docs/cmake-project-view.md

```diff
@@ -0,0 +1,124 @@
+# CMake Options Visibility Configuration
+
+Starting in CMake Tools 1.16, we will provide users the ability to customize their status bar with related CMake items like presets, kits, variants, build, configure, and launch to allow for users to de-clutter this global status bar as much as they would like to, while also having quick access to commonly performed CMake actions.
+
+Users can specify what is shown in the status bar through their `settings.json` file.
+
+The default settings will be set to the following:
+## Default Settings Json
+```json
+"cmake.options.statusBarVisibility": "hidden"
+"cmake.options.advanced": {
+    "build": {
+        "statusBarVisibility": "visible"
+    },
+    "launch": {
+        "statusBarVisibility": "visible"
+    },
+    "debug": {
+        "statusBarVisibility": "visible"
+    }
+}
+```
+
+These settings mean that by default all settings will be hidden from the status bar, except for Build, Launch, and Debug actions.
+
+To revert to the prior experience and have all of your presets display in the status bar, set your `Cmake > Options: Status Bar: Visibility` setting to `Visible`. This setting is overwritten by`CMake > Options: Advanced`, so if you had added to have some options hidden through this, these options will still remain hidden. 
```

Comment:
```suggestion
To revert to the prior experience and have all of your presets display in the status bar, set your `Cmake > Options: Status Bar Visibility` setting to `Visible`. This setting is overwritten by `CMake > Options: Advanced`, so if you had added to have some options hidden through this, these options will still remain hidden. 
```

Reply:
I learned how to make suggestions, so you don't have to copy paste lol. There was still a tiny typo here.

Reply:
If you like the suggestions I have, you can directly commit the changes. If you want me to edit my suggestions, I can do that too before you commit so you don't have to do that from the IDE

Reply:
It should be `CMake`, not Cmake

Reply:
@gcampbell-msft the setting itself says Cmake from some reason. @snehara99 is there any ability to change that?

Reply:
@gcampbell-msft @sinemakinci1 Yeah there's no way to change that